vnext/Shared/OInstance.cpp

```diff
@@ -425,6 +425,23 @@ InstanceImpl::InstanceImpl(
           m_turboModuleRegistry,
           m_innerInstance->getJSCallInvoker());
     } else {
+#if defined(USE_V8)
```

Comment:
>USE_V8 [](http://example.com/codeflow?start=12&length=6)

(nit) why would we want such flag? Can't we just make V8 default? Or we don't want to break existing behavior?

Reply:
This code is also shared with UWP, where V8 is not enabled